src/script/cpp_api/s_base.h

```diff
@@ -54,6 +54,8 @@ extern "C" {
 #define setOriginFromTable(index) \
 	setOriginFromTableRaw(index, __FUNCTION__)
 
+enum class ScriptingType: u8 { Client, Server, MainMenu};
```

Comment:
Missing space before the `}` is incorrect (or I think so, it's inconsistent with the `{` at least), but I'd just split this up over multiple lines.